front/app/components/UI/FormComponents/index.tsx

```diff
@@ -62,7 +63,7 @@ const FormSectionDescriptionStyled = styled.p`
 `;
 
 const StyledSpan = styled.span`
-  margin-right: 8px;
+  margin-right: 4px;
   font-weight: 600;
```

Comment:
4px looked better to me when the tooltip was included, and I still think the margin looks good when there isn't a tooltip icon :thinking: 

![image](https://github.com/CitizenLabDotCo/citizenlab/assets/33987955/54058044-9fa2-4e61-9e8f-ff7dd7590394)